### PR TITLE
2 new and 6 modified ruby advisories

### PR DESCRIPTION
**Pull request by jasnow**
2 new and 6 modified (my new approach) rubies/ruby advisories based on GHSA and ruby-advisory-db data.

yamllint and "rake" are green.

5 files did not make the cut.

### CODE REVIEW

File: rubies/ruby/CVE-2015-1855.yml [hunk 1]
```diff
@@ -11,7 +11,29 @@ description: |
...
+  (4) non-ASCII characters.
+cvss_v2: 4.3
+cvss_v3: 5.9
 patched_versions:
-- ~> 2.0.0.645
-- ~> 2.1.6
-- '>= 2.2.2'
+ - '~> 2.0.0-p645'
+ - '~> 2.1.6'
+ - '>= 2.2.2'
```
Comment (postmodern) [inline rubies/ruby/CVE-2015-1855.yml:26]: I thought we were standardizing on double-quotes for version ranges? Why single-quotes for these?

File: rubies/ruby/CVE-2019-16255.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 engine: ruby
 cve: 2019-16255
-url: https://www.ruby-lang.org/en/news/2019/10/01/code-injection-shell-test-cve-2019-16255/
-title: A code injection vulnerability of Shell#[] and Shell#test
+url: https://www.ruby-lang.org/en/news/2019/10/01/code-injection-shell-test-cve-2019-16255
+title: 'A code injection vulnerability of Shell#[] and Shell#test'
```
Comment (postmodern) [inline rubies/ruby/CVE-2019-16255.yml:5]: It doesn't appear that single-quotes are needed here:

```ruby
>> YAML.load("title: A code injection vulnerability of Shell#[] and Shell#tes
t\n")
=> {"title"=>"A code injection vulnerability of Shell#[] and Shell#test"}
```

File: rubies/ruby/CVE-2015-1855.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 engine: ruby
 cve: 2015-1855
-url: https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/
+url: https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability
```
Comment (postmodern) [inline rubies/ruby/CVE-2015-1855.yml:4]: It appears that `https://www.ruby-lang.org/` will redirect `https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability` back to `https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`. Better to keep using the canonical URL.

**Review by postmodern — CHANGES_REQUESTED** [review]
Noticed a few unnecessary changes.

File: rubies/ruby/CVE-2019-16255.yml (hunk 2 above)
Comment (jasnow) [reply rubies/ruby/CVE-2019-16255.yml:5]: will fix ; worried about []

File: rubies/ruby/CVE-2015-1855.yml (hunk 3 above)
Comment (jasnow) [reply rubies/ruby/CVE-2015-1855.yml:4]: both work

Comment (jasnow) [reply rubies/ruby/CVE-2015-1855.yml:4]: will fix

File: rubies/ruby/CVE-2015-1855.yml (hunk 1 above)
Comment (jasnow) [reply rubies/ruby/CVE-2015-1855.yml]: sync script does double-quotes and I see single-quotes and no quotes in the repo, so I pick singles.

**Comment by postmodern** [comment]
Cherry-picked 67b3c87 into master as a separate commit.

**Pull request closed without merging**